modules/cnf-deploying-the-du-infrastructure-profile.adoc

```diff
@@ -0,0 +1,269 @@
+// CNF-950 4.7 Deploying the DU infrastructure profile
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="scalability_and_performance/cnf-deploying-the-du-infrastructure-profile_{context}"]
+= Deploying the DU infrastructure profile
+
+[id="cnf-creating-the-performance-addon-operator-and-du-performance-profile_{context}"]
+== Creating the Performance Addon Operator and DU performance profile
+
+. Create the `kustomization.yaml` file, for example:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - performance-profile.yaml
+----
+
+. Create the `performance-profile.yaml` file, for example:
```

Comment:
```suggestion
. Create and apply the performance profile, for example:
```

Reply:
Change made.

Reply:
@vitus133 In the code bloc for this step, do you want to add the cat **<<EOF |  oc apply -f -**  & **EOF** statements?